gtfsscheduleviewer/files/index.js

```diff
@@ -11,25 +10,18 @@ function load(config) {
   forbid_editing = config.forbid_editing;
 
   sizeRouteList();
-  var map_dom = document.getElementById("map");
-  var mapOptions = {
-    zoom: 13,
-    center: config.bounds.getCenter(),
-    mapTypeId : google.maps.MapTypeId.ROADMAP,
-    scaleControl: true
-  };
-  map = new google.maps.Map(map_dom, mapOptions);
+
+  map = L.map('map')
+  L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
```

Comment:
Can you use our tiles? `https://cdn.mbta.com/osm_tiles/{z}/{x}/{y}.png` (attribution can stay the same).